packages/ui/src/pages/SignIn/index.tsx

```diff
@@ -1,22 +1,26 @@
+import React, { FC, FormEventHandler, useState, useCallback } from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
 import { signInBasic } from '@/apis/sign-in';
 import Button from '@/components/Button';
 import Input from '@/components/Input';
 import MessageBox from '@/components/MessageBox';
 import TextLink from '@/components/TextLink';
-import React, { FormEventHandler, useState } from 'react';
-import { useTranslation } from 'react-i18next';
+
 import styles from './index.module.scss';
 
 export type PageState = 'idle' | 'loading' | 'error';
 
-const Home = () => {
+const Home: FC = () => {
+  // TODO: Consider creading cross page data modal
```

Comment:
yeah. personally i would prefer [zustand](https://github.com/pmndrs/zustand). really neat and type friendly. it've been widely used in my current company.